config/conversion_issues.json

```diff
@@ -5,6 +5,7 @@
       "key": "a1",
       "active": true,
       "tool": "sakai.lessonbuildertool",
+      "actor": "LD",
```

Comment:
Do we want to make this a list ? so that if it is both or we add more variants then it can handle that.

Reply:
Addressed.